test/Npgsql.Tests/Replication/SafeReplicationTestBase.cs

```diff
@@ -139,7 +146,8 @@ async Task Cleanup()
             if (c.PostgreSqlVersion >= Pg10Version)
                 await c.ExecuteNonQueryAsync($"DROP PUBLICATION IF EXISTS {publicationName}");
 
-            await c.ExecuteNonQueryAsync($"DROP TABLE IF EXISTS {tableName}");
+            for (var i = tableNames.Length - 1; i >= 0; i--)
+                await c.ExecuteNonQueryAsync($"DROP TABLE IF EXISTS {tableNames[i]} CASCADE;");
```

Comment:
I wonder if we can use our "standard" tools for this (CreateTempTable etc.). But doesn't matter, out of scope in any case.

Reply:
Yeah probably.
I have to do some replication test infrastructure cleanup at some point.